en/2-relay-chain/2-reserve.md

```diff
@@ -1,25 +1,22 @@
-# Reserve a ParaID
+# Reserve a Parachain ID
```

Comment:
this is incorrect: ParaID could be a parachain _or_ thread.

Reply:
the changes should be propagated throughout the doc here

Reply:
Updated in the latest commit.